pkg/operator/controller/ingress/status.go

```diff
@@ -979,6 +987,16 @@ func updateIngressControllerAWSEIPAllocationStatus(ic *operatorv1.IngressControl
 	}
 }
 
+// updateIngressControllerFloatingIPOpenStackStatus mutates the provided IngressController object to
+// sync its status to the effective floatingIP on the LoadBalancer-type service.
+func updateIngressControllerFloatingIPOpenStackStatus(ic *operatorv1.IngressController, service *corev1.Service) {
+	// FloatingIP should be initialized by setDefaultPublishingStrategy
+	// when an IngressController is admitted, so we don't need to initialize here.
+	if fip := getFloatingIPFromSpec(service); fip != "" {
```

Comment:
Not quite, you'll want to get this from the service, the actual value. So you'll need a new function called `getLoadBalancerIPFromService`. Or you might not really even need the function, just read the value of `service.spec.loadBalancerIP`.

Reply:
I think I've addressed the comment.